shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/SoftForks.hs

```diff
@@ -0,0 +1,9 @@
+module Shelley.Spec.Ledger.SoftForks
+  ( validMetaData,
+  )
+where
+
+import Shelley.Spec.Ledger.PParams (PParams, PParams' (..), ProtVer (..))
+
+validMetaData :: PParams era -> Bool
+validMetaData pp = pvMinor (_protocolVersion pp) > 0
```

Comment:
Nice. For the future, we should probably document what these soft forks are here in comments.

Reply:
for sure, I meant to do that! I will add documentation next time I touch the code (and not now so that we can get this in). thank you for reminding me!